packages/edge/docs/README.md

```diff
@@ -36,7 +36,7 @@ City of the original client IP as calculated by Vercel Proxy.
 
 #### Defined in
 
-[src/edge-headers.ts:4](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L4)
+[packages/edge/src/edge-headers.ts:4](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L4)
```

Comment:
Not sure why these had to change. But the tests were failing and said to re run the docs:build script 🤷‍♂️